src/libponyc/codegen/gencall.c

```diff
@@ -1077,7 +1127,15 @@ LLVMValueRef gen_ffi(compile_t* c, ast_t* ast)
   pony_assert(t != NULL);
 
   // Get the function.
-  LLVMValueRef func = LLVMGetNamedFunction(c->module, f_name);
+  ffi_decl_t* ffi_decl;
+  bool is_func = false;
+  LLVMValueRef func = LLVMGetNamedGlobal(c->module, f_name);
+
+  if(func == NULL)
```

Comment:
Can you add a comment here that says in what cases the func will be retrieved from `LLVMGetNamedFunction` instead of `LLVMGetNamedGlobal`? I see that this case wasn't here before, so I'm curious what Pony types this corresponds to.

Reply:
If the name corresponds to a function, it will always be retrieved from `LLVMGetNamedFunction`. The `LLVMGetNamedGlobal` call is here to catch the cases where the name corresponds to an internal global and issue a compilation error for those cases.

I'll add a comment.

Reply:
Sorry, yeah, I had it backwards. Thanks for clearing it up for mw.